### DISCUSSION TITLE
Better Inline Code Styling

### DISCUSSION THREAD

**Discussion by bottle-rocket-11** [singerdmx/flutter-quill#848] [Feature Request]
Firstly, thank you for a great library. We have decided to use this library for the new version of our app running on all platforms, iOS, Android and Web. 

Inline code: we are trying to style our inline code similar to this screenshot below. However, it doesn't seem to be possible with any combination of styling offered. It would be awesome if we had full control of
- border
- background - partly works
- margin
- padding 
- textStyle - works

<img width="204" alt="image" src="https://user-images.githubusercontent.com/11312180/172205427-786b78ca-bf89-4c7d-9db4-7407382ded93.png">

It would also be AMAZING if when the curser is at the end of the inline code style, the right keyboard button concludes the inline code and creates a space ready for the normal styling to work again. Slack does it like that and I LOVE the function as I use it a lot

Best,
Hugo Pretorius

**Comment by singerdmx** [comment]
Just make a PR to change the default style for inline
